### DISCUSSION TITLE
Von OpenDTU OnBattery wieder auf die normale OpenDTU flashen? / Flashing from OpenDTU OnBattery back to the normal OpenDTU?

### DISCUSSION THREAD

**Discussion by Tobbz2k** [hoylabs/OpenDTU-OnBattery#986] [Q&A] [answered]
Ich habe eine OpenDTU gekauft, auf der schon die OpenDTU OnBattery Firmware drauf war. Kann ich hier einfach mit der OTA Funktion wieder auf die normale OpenDTU Version "zurück" flashen? Und wenn ja, welche Version muss ich dann nutzen?
------------------
I bought an OpenDTU that already had the OpenDTU OnBattery firmware on it. Can I simply flash back to the normal OpenDTU version using the OTA function? And if so, which version do I have to use?

**Comment by Tiese** [comment, marked as answer]
Hi @Tobbz2k,
geht prima (habe ich zu Testzwecken schon mehrfach -hin und her - gemacht).
Ich nehme immer die aktuelle Version.

Schöne Grüße
Tiese

**Comment by schlimmchen** [comment]
> welche Version muss ich dann nutzen

Schau in der Web UI unter Info -> System -> PIO Umgebung.

**Reply by d-blaue-Klaus** [reply]
Hallo Schlimmchen: da steht "generisch" ... wie komme ich damit weiter, um die korrekte Version zu finden?

**Comment by Tobbz2k** [comment]
Besten Dank, hat super geklappt!
